src/esp/nav/PathFinder.cpp

```diff
@@ -225,7 +225,7 @@ struct PathFinder::Impl {
              const float* bmax);
   bool build(const NavMeshSettings& bs, const esp::assets::MeshData& mesh);
 
-  vec3f getRandomNavigablePoint();
+  vec3f getRandomNavigablePoint(const int maxTries);
```

Comment:
```suggestion
  vec3f getRandomNavigablePoint(int maxTries);
```

it's just noise in declarations, makes sense only if the variable is actually used :)